pkg/webhook/utils.go

```diff
@@ -64,6 +64,58 @@ func labelsContainsLabel(labels []*github.Label, label string) bool {
 	return false
 }
 
+func doesPRNeedReview(pr *github.PullRequest, repo *github.Repository, gh *github.Client) (bool, error) {
+	reviewers, _, err := gh.PullRequests.ListReviewers(
```

Comment:
@chirino  This request only returns the reviewers which do not have yet approved (just tested it). So we can't use this method to find out whether an initially requested reviewer has approved as it doesnt show up here anymore.

We need to find another way to tackle this (e.g by finding a proper API call to get the initial list of requested reviewers or to store the reviewer ids as part of an HTML comment in the PR description (a bit evolved though).

However, thats the reason why the status checks always fails even when an initially requested reviewer has approved // cc. @gashcrumb 

A first solution would be to switch off this check again.

Reply:
Let me implement it like this: 

* If someone requests a reviewer, the label "status/review-requested" is added. 
  Then the "WIP" check (would probably rename it)
  * ... would be red if it's a WIP PR (titled or labeled as WIP)
  * ... would be red if the "status/review-requested" is given but no reviewer has approved
  * ... would be green if the label "status/review-requested" is not given
  * ... would be green if  the label "status/review-requested" is given and at least one reviewer (regardless who) has approved.
  
@deeleman @gashcrumb @chirino @zregvart would this fit our workflow ?

Reply:
who applies the `status/review-requested` label?  The bot or the person opening the PR?

Other than that question I say ship it :-)

Reply:
Its done by the bot automatically, as soon as a reviewer is requested. That way we can track also in retrospective the PRs for which reviewers were requested and which are unreviewed.

Reply:
@rhuss that sounds good to me.